docs/src/ncrings.md

```diff
@@ -259,6 +264,18 @@ Should returns `true` if parentheses are needed around this object when printed,
 a coefficient of a polynomial. As an example, non-constant polynomials would need such
 parentheses if used as coefficients of another polynomial.
 
+Note that since this approach quickly leads to unnecessary parentheses, the
+expression method below is preferred.
+
+### Expressions
+
+To obtain best results when printing composed types derived from other types, e.g., polynomials,
+the following method should be implemented
```

Comment:
```suggestion
the following method should be implemented.
```

Reply:
This leaves it unclear whether one needs to implement both `show` and `expressify`, and how they relate.

Looking at the AA source code, it seems that if one implements `expressify`, one can implement `show` as
```julia
function Base.show(io::IO, a::SOME_TYPE)
  print(io, AbstractAlgebra.obj_to_string(a, context = io))
end
```
If so, perhaps that could be stated?